.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
If CI runs for every push, why does it also need to run per pull request? For contributors?

Reply:
@hanzei Yes that's the idea

Reply:
Maybe I'm misunderstanding how the GitHub CI configuration works.

If a contributor forked the code in `master` and have a CI config with 
```yml
on:
  push:
    branches:
      - '**'
```

why is there a need to also add a trigger for `pull_request`?

Reply:
@hanzei Right, I think I goofed there. Let's make sure the shared GitHub action config is correct.

Reply:
Submitted https://github.com/mattermost/mattermost-app-google-drive/pull/13